developers.diem.com/README.md

```diff
@@ -38,20 +38,14 @@ Read our [Contributing guide](https://developers.diem.com/docs/community/contrib
 * [Diem: The Path Forward](https://developers.diem.com/blog/2019/06/18/the-path-forward/)
 
 ### Diem Codebase
-
-* [Diem Core Overview](https://developers.diem.com/docs/diem-core-overview)
-* [Admission Control](https://developers.diem.com/docs/crates/admission-control)
-* [Bytecode Verifier](https://developers.diem.com/docs/crates/bytecode-verifier)
-* [Consensus](https://developers.diem.com/docs/crates/consensus)
-* [Crypto](https://developers.diem.com/docs/crates/crypto)
-* [Execution](https://developers.diem.com/docs/crates/execution)
-* [Mempool](https://developers.diem.com/docs/crates/mempool)
-* [Move IR Compiler](https://developers.diem.com/docs/crates/ir-to-bytecode)
-* [Move Language](https://developers.diem.com/docs/crates/move-language)
-* [Network](https://developers.diem.com/docs/crates/network)
-* [Storage](https://developers.diem.com/docs/crates/storage)
-* [Virtual Machine](https://developers.diem.com/docs/crates/vm)
-
+* [Diem Core Overview](https://developers.diem.com/docs/core/overview)
+* [Bytecode Verifier](https://diem.github.io/diem/bytecode_verifier/index.html)
+* [Consensus](https://diem.github.io/diem/consensus/index.html)
+* [Crypto](https://diem.github.io/diem/diem_crypto/index.html)
+* [Mempool](https://diem.github.io/diem/diem_mempool/index.html)
+* [Move IR Compiler](https://diem.github.io/diem/ir_to_bytecode/index.html)
+* [Network](https://diem.github.io/diem/network/index.html)
+* [Virtual Machine](https://diem.github.io/diem/vm/index.html)
```

Comment:
I think we are not in sync here These should be removed. I was just talking about the specific removal you did starting on line 66.